configs/components/leatherman.rb

```diff
@@ -15,7 +15,11 @@
     pkg.build_requires "pl-cmake"
   elsif platform.is_aix?
     pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/aix/#{platform.os_version}/ppc/pl-cmake-3.2.3-2.aix#{platform.os_version}.ppc.rpm"
-    pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/aix/#{platform.os_version}/ppc/pl-boost-1.58.0-1.aix#{platform.os_version}.ppc.rpm"
+    if platform.name =~ /aix-7.1/
```

Comment:
Is this package also used on AIX 7.2?

Reply:
Sorry, that was a typo. I've updated the commit header and PR title to reflect that it's for aix 7.1.

Reply:
Do we need separate code to support AIX 7.2? Because that also appears to be failing: https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent/view/master/view/Non-Standard/job/platform_puppet-agent_intn-van-sys_suite-secondary-pxp-agent-master/10/SLAVE_LABEL=beaker,TEST_TARGET=aix72-POWERfa/testReport/(root)/tests_tasks/task_download_rb/

Reply:
No, I just confirmed with Branan that the packages we build for aix 7.1 are what we use for 7.2 targets as well. In the p-a platform list you'll see there is only a file for aix 7.1, so this code should work as intended.